config/application.rb

```diff
@@ -25,5 +25,8 @@ class Application < Rails::Application
     # config.generators do |g|
     #   g.test_framework :minitest
     # end
+    # config.active_record.encryption.primary_key = ENV['PRIMARY_KEY']
+    # config.active_record.encryption.deterministic_key = ENV['DETERMINISTIC_KEY']
+    # config.active_record.encryption.key_derivation_salt = ENV['KEY_DERIVATION_SALT']
```

Comment:
Could these be deleted?
```suggestion
```

Reply:
Yes I believe so. I think Rachal added these lines, but they're already in the encrypted creds